zio/zngio/scanner.go

```diff
@@ -78,6 +78,7 @@ func (s *zngScanner) scanUncompressed() ([]*zng.Record, error) {
 			return nil, err
 		}
 		if rec != nil {
+			rec.CopyBody()
```

Comment:
Both calls to scanOne are now followed by this call, can it move inside scanOne?

Reply:
Done in 0c0ffd0d07f19518a61585c514577572c269d786.